quinn/src/connection.rs

```diff
@@ -127,6 +127,17 @@ where
                     .expect("spurious handshake data ready notification")
             })
     }
+
+    /// The local IP address which was used when the peer established
+    /// the connection
+    ///
+    // This is `None` if either not known or if this is the client side of a connection.
```

Comment:
Was it intended that the last line here not be a doc comment?

Reply:
No. Will fix